pkg/favirecon/db.json

```diff
@@ -606,6 +609,7 @@
     "-986678507": "ISP Manager",
     "-986816620": "OpenRG",
     "-991123252": "VMware Horizon",
+    "203612613": "Login",
```

Comment:
This is `Request Tracker - Login`

Reply:
Fixed this in [525da92](https://github.com/edoardottt/favirecon/pull/171/commits/525da92d772675eaa05db2664f17beef066ff947).